config/cfg.py

```diff
@@ -15,6 +15,9 @@ def schema_violations_from_proposed_config(config: Dict) -> List[str]:
     for error in validator.iter_errors(config):
         schema_violation_messages.append(f"{error.message}")
 
+    if not isinstance(config, dict):
+        raise ValueError("Sweep config must be parsable as a JSON object.")
```

Comment:
why isnt this line covered my the included test?

Reply:
bad test. fixed